### PR TITLE
init fzf at 0.9.13

### PR DESCRIPTION
**Pull request by magnetophon**

### CODE REVIEW

File: pkgs/tools/misc/fzf/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  };
+
+  buildInputs = with goPackages; [ crypto ginkgo gomega go-runewidth go-shellwords ncurses syncthing text ];
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/junegunn/fzf;
+    description = "A command-line fuzzy finder written in Go";
+    license = licenses.mit;
+    maintainers = [ maintainers.magnetophon ];
+    platforms = ["x86_64-linux"];
```
Comment (pSub) [inline pkgs/tools/misc/fzf/default.nix]: Why only that platform?

**Comment by magnetophon** [comment]
@ps

File: pkgs/top-level/go-packages.nix [hunk 2]
```diff
@@ -1230,6 +1230,30 @@ let
...
+      owner = "junegunn";
+      repo = "go-runewidth";
+      sha256 = "07d612val59sibqly5d6znfkp4h4gjd77783jxvmiq6h2fwb964k";
+    };
+  };
+
+  go-shellwords = buildGoPackage rec {
+    rev = "35d512af75e283aae4ca1fc3d44b159ed66189a4";
+    name = "go-shellwords-${rev}";
+    goPackagePath = "github.com/junegunn/go-shellwords";
```
Comment (magnetophon) [inline pkgs/top-level/go-packages.nix]: This is actually forked from https://github.com/mattn/go-shellwords, so maybe I should put the junegunn version under a different name?
Though the original has been last updated 2 years ago...

File: pkgs/top-level/go-packages.nix [hunk 3]
```diff
@@ -1230,6 +1230,30 @@ let
     };
   };
 
+  go-runewidth = buildGoPackage rec {
+    rev = "travisish";
+    name = "go-runewidth-${rev}";
+    goPackagePath = "github.com/junegunn/go-runewidth";
```
Comment (magnetophon) [inline pkgs/top-level/go-packages.nix]: This is actually forked from https://github.com/mattn/go-runewidth, so maybe I should put the junegunn version under a different name?

Comment (lethalman) [inline pkgs/top-level/go-packages.nix]: `junegunn.go-runewidth`

**Comment by matthiasbeyer** [comment]
Nice to have fzf finally in the nixos packages, but _please_ split up the commit, so you have one commit for each package you add.

With several commits, you could also follow the new contributing styleguide and name them "<package>: init at X.Y"!

**Comment by magnetophon** [comment]
improved PR at https://github.com/NixOS/nixpkgs/pull/8731

**Pull request closed without merging**